projects/angular/src/layout/nav/_header.clarity.scss

```
@@ -588,95 +344,48 @@
         background-repeat: no-repeat;
         background-size: contain;
         cursor: pointer;
-        @include mixins.min-equilateral-var(
-          clr-header-search-icon-size,
-          header-variables.$clr-search-icon-width,
-          variables.$clr-use-custom-properties
-        );
-        margin: variables.$clr_baselineRem_0_83 0 0;
-        vertical-align: top;
-        @include mixins.css-var(
-          margin-right,
-          clr-header-search-icon-margin-right,
-          0,
-          variables.$clr-use-custom-properties
+        @include mixins.min-equilateral(header-variables.$clr-header-search-icon-size);
+        vertical-align: middle;
+        margin: 0;
+        margin-right: calc(
+          -1 * (header-variables.$clr-header-search-icon-size + header-variables.$clr-header-search-gap)
         );
       }
 
       input {
-        line-height: variables.$clr_baselineRem_1;
-        @include mixins.css-var(
-          margin,
-          clr-header-search-margin,
-          header-variables.$clr-header-nav-text-vertical-padding 0,
-          variables.$clr-use-custom-properties
-        );
+        line-height: tokens.$cds-global-space-9;
+        margin: auto;
       }
     }
 
     input[type='text'] {
       border: none;
       background: none;
-      @include mixins.css-var(
-        color,
-        clr-header-search-input-font-color,
-        header-variables.$clr-header-font-color,
-        variables.$clr-use-custom-properties
-      );
-
-      @include mixins.css-var(
-        height,
-        clr-header-search-height,
-        variables.$clr_baselineRem_1,
-        variables.$clr-use-custom-properties
-      );
+      color: header-variables.$clr-header-search-input-font-color;
+
+      height: header-variables.$clr-header-search-height;
```

Missing `border-radius` from design specification.

```suggestion
      color: header-variables.$clr-header-search-input-font-color;
      border-radius: tokens.$cds-alias-object-border-radius-100;
      height: header-variables.$clr-header-search-height;
```

---

Yes, better now, added.